src/ocispec/ocijsonspec.hpp

```diff
@@ -66,8 +66,10 @@ static const struct json_obj_descr RuntimeSpecJsonDescr[] = {
 
 struct ImageConfig {
     char*           Cmd[aos::oci::cMaxParamCount];
```

Comment:
Could we have len just below the corresponding array?

Reply:
done